cohort_3/curriculum/workshops/maestro-website/README.md

```diff
@@ -0,0 +1,110 @@
+# Maestro + webpage workshop
+
+## Website bootstrap
+
+- Open a cloud9 workspace and create a new folder titled `texting-site`
+- Inside the folder create a new file called `index.html`
+- Make a basic website template with a `<head>` and `<body>`
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+
+  </head>
+  <body>
+
+  </body>
+</html>
+```
+
+- Inbetween `<head>` tags, add `<script></script>` tags with an attribute `src`
+  that is equal to `https://maestro.ngrok.com/static/baton.js`
+- Open your website, right click on the page, click 'inspect element', and click
+  'console'
+  - If there are errors, change `https` => `http` in
+    the URL 
+- In the console, type in `maestro.Twilio.sendSms('415-555-5555','How is your
+  day?')`, replacing the phone number with your own phone number
+- Below the `<script>` tag add a second `<script>` tag in your `<head>` with the
+  `src="script.js"`
+- Create a new file in Cloud9 and name it `script.js`
+
+## Texting with programming for fun and profit
+
+### Input boxes
+
+- In the body, add two `<input type="text">` tags (one for recipient phone number, one for
+  message)
+  - `<input>` tags are self-closing, so don't add a closing `</input>` to the
+    end
+- Give one input an `id` attribute equal to `phone-number` and the other input
+  an `id` of `message`
+
+### Button that does something when clicked
+
+- Under the two inputs we created, create a `<button>Click me</button>`
+- Give the button an `onclick` attribute equal to `sendText();`
+- In `script.js`, add the following:
+
+```js
+var sendText = function () {
+  alert('button pressed!');
+};
+```
+
+- Press the button on your website
+  - Anything you put inside the `sendText` function will be run when you click
+    the button
+
+### Something that sends a text when button is pressed
+
+- In the `sendText` function in `script.js` add
+  `maestro.Twilio.sendSms('415-555-5555', 'You pressed a button!')` replacing
+  the `415-555-5555` number with your own phone number
+- Press the button.
+
+### Function that takes number and message from input boxes
+
+- Open your browser console by right clicking on your website, clicking inspect
+  element, and clicking `console`
+- Type `document.getElementById('phone-number')`
+- Type `document.getElementById('phone-number').value`
+    - If you press the up arrow when in console it will go back to the last
+      command you entered, so you don't need to retype it all
```

Comment:
Should be 2 space indent. The `markdown-mode` package for Emacs is annoying and doesn't let you adjust the space increment – I'm thinking about just creating a fork myself that allows customizing that.